components/WheelContainer/WheelContainer.js

```diff
@@ -14,67 +13,57 @@ const StyledSlotWrapper = styled.div`
 `;
 
 const StyledSlot = styled.div`
+  font-size: 150%;
   display: flex;
-  height: 200px;
+  height: 240px;
   width: 6rem;
   flex-direction: column;
   align-items: center;
-  gap: 10px;
   border: 1px solid #bababa;
   border-radius: 5px;
   background-color: #f8f8f8;
-  @media (min-width: 400px) {
-    font-size: 150%;
+  @media (min-width: 414px) {
+    font-size: 175%;
     border-radius: 30px;
     background-color: #f8f8f8;
     display: flex;
-    width: 80%;
+    width: 7.5rem;
     height: 280px;
     align-items: center;
-    margin-bottom: 50px;
     border: 1px solid #bababa;
     border-radius: 5px;
   }
   @media (min-width: 700px) {
-    font-size: 250%;
-    border-radius: 30px;
-    background-color: #f8f8f8;
-    display: flex;
-    width: 80%;
-    height: 440px;
-    align-items: center;
-    margin-bottom: 50px;
-    border: 2px solid #bababa;
-    border-radius: 5px;
+    font-size: 350%;
+    width: 9.5rem;
+    height: 380px;
   }
 `;
 
+const StyledParagraph = styled.div`
```

Comment:
Since it represents a paragraph, should `<p>` perhaps be considered instead of `<div>`? 
Unless you don't want that thing where the browser automatically adds an extra blank line before & after `<p>`, of course